qemu/tests/cfg/qsd_backup_push.cfg

```diff
@@ -0,0 +1,47 @@
+- qsd_backup_push:
+    type = qsd_backup_push
+    virt_test_type = qemu
+    images += " nbd1"
+    start_vm = no
+    storage_pools = default
+    storage_pool = default
+    storage_type_default = "directory"
+
+    image_size_stg1 = 2G
+    image_size_base = 2G
+    image_size_inc = 2G
+
+    image_format_stg1 = qcow2
+    image_format_base = qcow2
+    image_format_inc = qcow2
+
+    image_name_stg1 = stg1
+    image_name_base = base
+    image_name_inc = inc
+    source_images = "stg1"
+    image_backup_chain_stg1 = "base inc"
+    rebase_mode = unsafe
+    only Linux
+    qemu_force_use_drive_expression = no
+    blk_extra_params_stg1 = "serial=DATA_DISK1"
+    blk_extra_params_base = ${blk_extra_params_stg1}
+    blk_extra_params_inc = ${blk_extra_params_stg1}
```

Comment:
We can remove these 3 lines, for these params cannot take effect for the vhost-user-blk-pci device, so in the debug log you can see that DAT_DISK1 cannot be found. 
The test result passed because we use the image size to find out the data disk.